samples/matter/light_bulb/child_image/mcuboot/prj.conf

```diff
@@ -18,3 +18,6 @@ CONFIG_SERIAL=n
 CONFIG_UART_CONSOLE=n
 CONFIG_USE_SEGGER_RTT=n
 CONFIG_GPIO=n
+
+# Use minimal C library instead of the Picolib
```

Comment:
Why is it necessary? Matter used newlib so far and (at least) in the past minimal libc was not enough to build and run Matter stack successfully.

Reply:
Are you sure it applies to mcuboot as well? The changes are required only for the mcuboot image and I think we currently use minimal libc, co basically nothing should change. This is necessary, because something changed in the upstream mcuboot and they started to use different libc, so we are not able to fit within our mcuboot partition.

Reply:
> Why is it necessary? Matter used newlib so far and (at least) in the past minimal libc was not enough to build and run Matter stack successfully.

Will bring back newlib in your case, but if that fails: next fix is no you.

Reply:
> > Why is it necessary? Matter used newlib so far and (at least) in the past minimal libc was not enough to build and run Matter stack successfully.
> 
> Will bring back newlib in your case, but if that fails: next fix is no you.

No wait. 

> Are you sure it applies to mcuboot as well? The changes are required only for the mcuboot image and I think we currently use minimal libc, co basically nothing should change. This is necessary, because something changed in the upstream mcuboot and they started to use different libc, so we are not able to fit within our mcuboot partition.

If that is true, then min lib C should suffice. Otherwise you are getting picolib and that makes MCUboot bigger.

Reply:
right, my bad. Didn't notice it's for mcuboot...